TKGame/Components/Concrete/Item_PhysicsComponent.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TKGame.Components.Concrete;
+using TKGame.Components.Interface;
+using TKGame.BackEnd;
+using Myra.Graphics2D.UI;
+using System.Drawing;
+
+namespace TKGame.Components.Concrete
+{
+    internal class Item_PhysicsComponent : PhysicsComponent
+    {
+        void PhysicsComponent.Update(Entity entity, GameTime gameTime/*, World &world*/)
+        {
+            Player player = Player.Instance;
+
+            // if player is within pickup range of item
```

Comment:
Don't know if there is supposed to be code here, but I don't see any pickup functionality for the items. Might just not be in the PR.

Reply:
Haven't tested code, so I don't know if it is functional.